arrow-buffer/src/util/bit_chunk_iterator.rs

```diff
@@ -212,6 +219,7 @@ pub struct BitChunks<'a> {
 }
 
 impl<'a> BitChunks<'a> {
+    /// Create a new [`BitChunks`] from a byte array, and and an offset and length in bits
```

Comment:
Thank you -- the fact that the length is in bits I think is real value add to this documentation

```suggestion
    /// Create a new [`BitChunks`] from a byte array, and an offset and length in bits
```